docs/manual/gettingStarted/helpAndTroubleshooting/troubleshooting.md

```diff
@@ -11,36 +11,38 @@ Startup crashes or freezes usually happen because of these reasons:
 
-2) Caused by some 3rd party. This is usually:
+2) Caused by a third party, typically by:
 
-- extension or theme. In this case Playnite tries to detect if a crash was caused by an add-on and will show appropriate error about it, but this is always not possible. You can test if an add-on is causing crash by starting Playnite in safe mode (from Help menu or via `Safe Mode.bat` from Playnite's installation folder), or by manually disabling extensions (from add-ons menu) until the crash goes away.
+- **Extension or theme:**
+  
+  In this case Playnite tries to detect if a crash was caused by an add-on and will show appropriate error about it, but this is always not possible. You can test if an add-on is causing crash by starting Playnite in safe mode (from Help menu or via `Safe Mode.bat` from Playnite's installation folder), or by manually disabling extensions (from add-ons menu) until the crash goes away.
 
-- 3rd party application or system malfunction. This is usually true for crashes that just close Playnite without any crash dialog or error message whatsoever. Usually caused by applications that inject overlays or interact with GPU accelerated apps (and games) in some other manner. Also sometimes caused by anti-virus applications falsely flagging Playnite and deleting/blocking its files.
+- **3rd party application or system malfunction:**
+  
+  This is usually true for crashes that just close Playnite without any crash dialog or error message whatsoever. Usually caused by applications that inject overlays or interact with GPU accelerated apps (and games) in some other manner. Also sometimes caused by anti-virus applications falsely flagging Playnite and deleting/blocking its files.
 
-Applications recently known to cause most issues:
-- Avast
-- AVG
-- Malwarebytes
-- Nahimic service
+  Applications recently known to cause most issues:
+  - Avast
+  - AVG
+  - Malwarebytes
+  - Nahimic service
 
 ### General network issues
   
 Usually occurs while installing Playnite or downloading addons. It can happen due to multiple reasons:
   
-  - Firewall or anti-virus blocking Playnite processes
-  - DNS functionality being broken on ISP's side
-  - [network issue on ISP's side](https://github.com/JosefNemec/Playnite/issues/3268)
-  - restarting internet modem also helped some people to fix network related issues
-  
-  In case of DNS issues, you can use CloudFlare's or Google's DNS servers:
-  
-https://developers.cloudflare.com/1.1.1.1/setup/windows/
+  - Firewall or anti-virus blocking Playnite processes.
+  - A [network issue on your ISP's (Internet Service Provider) side](https://github.com/JosefNemec/Playnite/issues/3268).
+  - Some users have reported that restarting their internet modem helped resolve network-related problems.
 
-https://developers.google.com/speed/public-dns/docs/using#windows
+If you suspect DNS issues, you can switch to using either CloudFlare's or Google's DNS servers:
```

Comment:
> If you suspect DNS issues

Most people will have no idea what DNS is, so they will unlikely suspect it being that.

Reply:
Changes made in https://github.com/JosefNemec/PlayniteDocs/pull/9/commits/d23fe6804fa49dd049c3845ed1331116ab9be163

I added a small and simple explanation of what DNS is and why it could cause issues.